sqlite/src/parse.y

```diff
@@ -313,9 +313,9 @@ columnname(A) ::= nm(A) typetoken(Y). {sqlite3AddColumn(pParse,&A,&Y);}
   ADD AGGREGATE ALIAS ANALYZEEXPERT ANALYZESQLITE AUTHENTICATION
   BLOBFIELD BULKIMPORT
   CHECK COMMITSLEEP CONSUMER CONVERTSLEEP COUNTER COVERAGE CRLE
-  DATA DATABLOB DATACOPY DBPAD DEFERRABLE DISABLE DISTRIBUTION DRYRUN
-  ENABLE EXEC EXECUTE FUNCTION GENID48 GET GRANT INCREMENT IPU ISC KW
-  LUA LZ4 NONE
+  DATA DATABLOB DATACOPY DBPAD DEFERRABLE DETERMINISTIC DISABLE 
```

Comment:
Do we want to keep "deterministic"?  It's just an assertion that we can't verify.

Reply:
@mponomar Yes, the flag `SQLITE_DETERMINISTIC` is passed while creating `sfunc` according to if `deterministic` was specified at the creation time. If `SQLITE_DETERMINISTIC` is not specified, then the user wouldn't be able to use it in the table schema. On the other hand, I can also not mark every `scalar function` to be `deterministic` and pass the `SQLITE_DETERMINISTIC` flag. So, I kind of rely on the user to tell me with the "deterministic" keyword when to pass the flag.